source/docs/software/vision-processing/apriltag/apriltag-intro.rst

```diff
@@ -0,0 +1,233 @@
+.. include:: <isonum.txt>
+
+What Are AprilTags?
+===================
+
+.. image:: images/apriltagrobots_overlay.png
+
+AprilTags are a system of visual tags developed by researchers at the University of Michigan to provide low overhead, high accuracy localization for many different applications.
+
+Application to FRC
+------------------
+
+In the context of FRC, AprilTags are useful for helping your robot know where it is at on the field, so it can align itself to some goal position.
+
+AprilTags have been in development since 2011, and have been refined over the years to increase the robustness and speed of detection.
+
+Starting in 2023, FIRST is providing a number of tags, scattered throughout the field, each at a known :term:`pose`.
+
+All of the tags are from the 36H11 family, which means that:
+
+1) Each tag carries 36 bits of information
+2) It would take at least 11 bit "flips" before one tag could be mistaken for another.
```

Comment:
Is this necessary? Are tag bit flips more common than typical bit flips?

Reply:
Could be removed or put in a collapsed section, just explaining what each number in the 36h11 means